examples/openspending/components/_shared/Footer.tsx

```diff
@@ -0,0 +1,13 @@
+import Image from 'next/image'
+export default function Footer() {
+  return (
+    <footer>
+      <div className="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8 flex flex-col items-center justify-between md:flex-row">
+        <Image alt="Datopian logo" src="/datopian-logotype.png" width={160} height={40} />
```

Comment:
Image should also link to the Datopian website

Reply:
Done